src/website.py

```diff
@@ -0,0 +1,102 @@
+from os import path
+
+from flask import Flask
+from openai.embeddings_utils import cosine_similarity
+
+from compute_embedding import embedding_from_string
+
+app = Flask(__name__)
+
+@app.route('/')
+def home():
+    return '''<form>
+    <label for="fname">Enter first file file: </label><br>
```

Comment:
file file -> file name
This makes sense under limited use cases only. Usually the server and client are different computers.
The server must be able to run Python, with the special libraries and OPEN_API_KEY. The client is just some computer with a browser. Only in this limited development setting the server and client are the same computer. 
The file must be on the server in a particular directory.